src/test/java/com/puppycrawl/tools/checkstyle/checks/header/RegexpHeaderCheckTest.java

```diff
@@ -24,13 +24,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.times;
-import static org.powermock.api.mockito.PowerMockito.doNothing;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.verifyStatic;
```

Comment:
Is this the same here or do we need the class annotations because of `Whitebox`?